commit/core.py

```diff
@@ -84,6 +84,8 @@ def load_data( self, dwi_filename = 'DWI.nii', scheme_filename = 'DWI.scheme', b
         self.niiDWI  = nibabel.load( pjoin( self.get_config('DATA_path'), dwi_filename) )
         self.niiDWI_img = self.niiDWI.get_data().astype(np.float32)
         hdr = self.niiDWI.header if nibabel.__version__ >= '2.0.0' else self.niiDWI.get_header()
+        if hdr['dim'][0] == 3:
+            self.niiDWI_img = np.expand_dims(self.niiDWI_img,3)
```

Comment:
Just double check that we won't use anymore the field `hdr['dim'][0]`. Otherwise, we probably need to overwrite it as `hdr['dim'][0] = 4` for consistency